osquery/dispatcher/scheduler.h

```diff
@@ -33,15 +35,19 @@ class SchedulerRunner : public InternalRunnable {
   /// The Dispatcher interrupt point.
   void stop() override {}
 
+  std::chrono::milliseconds getTimeDrift() const noexcept;
```

Comment:
nit: given this is not accumulated drift anymore please reflect that in the name, something like getCurrentTimeDrift for example.

Reply:
got it, done